coalib/coala.py

```diff
@@ -33,7 +33,51 @@ def main():
         args = default_arg_parser().parse_args()
         console_printer = ConsolePrinter(print_colored=not args.no_color)
 
-        if args.show_bears:
+        if args.json:
```

Comment:
can we have the specialized "main functions" in own functions at least? Possibly in own files even ut not sure about that